cmd/dep/gb_importer.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/dep"
+	fb "github.com/golang/dep/internal/feedback"
+	"github.com/golang/dep/internal/gps"
+	"github.com/pkg/errors"
+)
+
+// gbImporter imports gb configuration into the dep configuration format.
+type gbImporter struct {
+	manifest gbManifest
+	logger   *log.Logger
+	verbose  bool
+	sm       gps.SourceManager
+}
+
+func newGbImporter(logger *log.Logger, verbose bool, sm gps.SourceManager) *gbImporter {
+	return &gbImporter{
+		logger:  logger,
+		verbose: verbose,
+		sm:      sm,
+	}
+}
+
+// gbManifest represents the manifest file for GB projects
+type gbManifest struct {
+	Version      int            `json:"version"`
+	Dependencies []gbDependency `json:"dependencies"`
+}
+
+type gbDependency struct {
+	Importpath string `json:"importpath"`
+	Repository string `json:"repository"`
+
+	// All gb vendored dependencies have a specific revision
+	Revision string `json:"revision"`
+
+	// Branch may be HEAD or an actual branch. In the case of HEAD, that means
+	// the user vendored a dependency by specifying a tag or a specific revision
+	// which results in a detached HEAD
+	Branch string `json:"branch"`
+
+	// Path is used when the actual used package exists in a subpath
+	// For importing purposes, this isn't necessary as dep will figure
+	// out which packages are actually being used during resolving
+	Path string `json:"path,omitempty"`
```

Comment:
Since we are ignoring `Path`, and don't print a warning about the fact that we ignore it, we can just remove it. 😄